acceptance/suites/tests/010-puppetserver-cli/subcommand/foreground.rb

```diff
@@ -0,0 +1,44 @@
+test_name "Puppetserver 'foreground' subcommand tests."
+
+# --------------------------------------------------------------------------
+# This behavior needs to be baked into Beaker but is not yet. SERVER-79 tracks
+# the need to fix this once Beaker has been modified to make the paths to these
+# commands available.
+#
+if master.is_pe?
+  cli = "pe-puppetserver"
+else
+  cli = "puppetserver"
+end
+# --------------------------------------------------------------------------
+
+# puppetserver seems to take about 45s to start up
+timout_length = "60s"
```

Comment:
60s is probably okay since the run that generates SSL certs should happen in the pre-suite - 90_validate_sign_cert.rb.  We've been using 120 seconds as the `TimeoutStartSec` for the startup of the service without any problems in CI - although that is only gated to the webserver coming to ready, which would happen after 1 JRubyPuppet instance has been initialized but not necessarily any others.